### PR TITLE
Add tqdm to `rb.log`

### PR DESCRIPTION
**Pull request by dcfidalgo**
This PR adds a progress bar to the `rb.log` method, and also prints out the URL where the dataset can be found.
I left a todo to improve our logging policy in the library, for now, it is a simple `print` ...

### CODE REVIEW

File: src/rubrix/client/rubrix_client.py [hunk 1]
```diff
@@ -200,6 +205,12 @@ def log(
             processed += response.parsed.processed
             failed += response.parsed.failed
 
+            progress_bar.update(len(chunk))
+        progress_bar.close()
+
+        # TODO: improve logging policy in library
+        print(f"Records logged to {self._client.base_url + '/' + name}")
```
Comment (dvsrepo) [inline src/rubrix/client/rubrix_client.py]: Can we add the number of records: print(f"{len(processed} records logged to {self._client.base_url + '/' + name}")

**Review by dvsrepo — CHANGES_REQUESTED** [review]

File: src/rubrix/client/rubrix_client.py (hunk 1 above)
Comment (dcfidalgo) [reply src/rubrix/client/rubrix_client.py]: por supuesto, let's just keep in mind that this info is also present in the `BulkResponse`.

**Comment by frascuchon** [comment]
Closes #410 ?

**Comment by dcfidalgo** [comment]
Yesss, closes #410 !

**Review by dvsrepo — APPROVED** [review]

**Pull request merged**